tests/unit/routes/workflows/versions/index-test.js

```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('route:workflows/versions/index', 'Unit | Route | workflows/versions/index', {
+  // Specify the other units that are required for this test.
+  // needs: ['controller:foo']
+});
+
+test('it exists', function(assert) {
```

Comment:
I know many of the route tests are empty/trivial, but since this route has some dependencies on its nesting, it's probably good to test it in case we ever move it around.

Reply:
Not quite sure how to test this. The emberjs guide is focused on testing actions: https://guides.emberjs.com/v2.14.0/testing/testing-routes/

Reply:
> Acceptance tests will likely provide better coverage for routes because routes are typically used to perform transitions and load data, both of which are tested more easily in full context rather than isolation.

I think they don't automatically generate acceptance tests for you, since they're larger in scope and have no obvious defaults per route.

But here, we do have a unit test that could test the functionality we have - looking up the workflow id and querying for versions based on that. The unit test could simply instantiate the route with a mocked store (and params), and assert that it the store is called with the workflow id in the params
```